include/zephyr/net/net_if.h

```diff
@@ -1504,6 +1504,10 @@ uint32_t net_if_ipv6_calc_reachable_time(struct net_if_ipv6 *ipv6);
 static inline void net_if_ipv6_set_reachable_time(struct net_if_ipv6 *ipv6)
 {
 #if defined(CONFIG_NET_NATIVE_IPV6)
+	if (!ipv6) {
```

Comment:
One small question though. I've seen [other reviews](https://github.com/zephyrproject-rtos/zephyr/pull/53041#discussion_r1050432427) ask to make the NULL pointer check explicit with

```suggestion
	if (ipv6 != NULL) {
```

Is this preferred, is this documented?

Reply:
I think it's MISRA requirement to be explicit, but good point, I'll update the check.

Reply:
Not related to this PR, but the file has other NULL pointer checks which aren't MISRA compliant (probably a lot in of these in the zephyr tree).